masterbase/app.py

```diff
@@ -166,31 +166,30 @@ async def on_accept(self, socket: WebSocket, api_key: str, session_id: str) -> N
 
         await set_open_true(engine, steam_id, session_id)
 
-        path = make_demo_path(session_id)
+        session_manager = DemoSessionManager(session_id=session_id, detection_state=DetectionState())
```

Comment:
I like this motion to a manager paradigm - would it be possible to use Pythonic handlers in combination here too? Anything to avoid explicit `.close()` function calls lol. + the Handler syntax is nice for being fairly explicit with the control flow.